samcli/lib/remote_invoke/remote_invoke_executors.py

```diff
@@ -177,10 +207,12 @@ def __init__(
     def execute(self, remote_invoke_input: RemoteInvokeExecutionInfo) -> RemoteInvokeExecutionInfo:
         """
         First runs all mappers for request object to get the final version of it.
-        Then invokes the BotoActionExecutor to get the result
+        Then validates all the input boto parameters and invokes the BotoActionExecutor to get the result
         And finally, runs all mappers for the response object to get the final form of it.
         """
         remote_invoke_input = self._map_input(remote_invoke_input)
+        remote_invoke_output = None
```

Comment:
Just curious, do we need to define `remote_invoke_output` as `None` here, can't we just let line 216 to set it?

Reply:
Nope it doesn't need to be defined. Addressed in the new commit.